### PR TITLE
Update compliance-manager-templates-format-excel.md

### PR DESCRIPTION
**Pull request by msbemba**
Added sub bullets under Actions/implementationType

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7822

File: microsoft-365/compliance/compliance-manager-templates-format-excel.md [hunk 1]
```diff
@@ -76,10 +76,10 @@ The **Actions** tab is required.  It designates improvement actions managed by y
...
+- **implementationType**: In this required field, list one of the three implementation types below: 
+  1) **Operational** - actions implemented by people and processes to protect the confidentiality, integrity, and availability of organizational systems, assets, data, and personnel (example: security awareness and training)
+  2) **Technical** - actions completed by using technology and mechanisms contained in the hardware, software, or firmware components of the information system to protect the confidentiality, integrity, and availability of organizational systems and data (example: multi-factor authentication)
+  3) **Documentation** - actions implemented through documented policies and procedures establishing and defining the controls required to protect the confidentiality, integrity, and availability of organizational systems, assets, data, and personnel (example: an information security policy)
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/compliance-manager-templates-format-excel.md]:
```suggestion
- **implementationType**: In this required field, list one of the following three implementation types: 
  1) **Operational** - actions implemented by people and processes to protect the confidentiality, integrity, and availability of organizational systems, assets, data, and personnel (example: security awareness and training).      
  2) **Technical** - actions completed by using technology and mechanisms contained in the hardware, software, or firmware components of the information system to protect the confidentiality, integrity, and availability of organizational systems and data (example: multi-factor authentication).
  3) **Documentation** - actions implemented through documented policies and procedures establishing and defining the controls required to protect the confidentiality, integrity, and availability of organizational systems, assets, data, and personnel (example: an information security policy).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**